sdk/servicebus/Azure.Messaging.ServiceBus/tests/Processor/ProcessorTests.cs

```diff
@@ -125,6 +125,7 @@ public void ProcessorOptionsSetOnClient()
             var fullyQualifiedNamespace = new UriBuilder($"{account}.servicebus.windows.net/").Host;
             var connString = $"Endpoint=sb://{fullyQualifiedNamespace};SharedAccessKeyName=RootManageSharedAccessKey;SharedAccessKey={Encoding.Default.GetString(ServiceBusTestUtilities.GetRandomBuffer(64))}";
             var client = new ServiceBusClient(connString);
+            var identifier = "MyProcessor";
```

Comment:
Let's do an equivalent update to SessionProcessorTests.

Reply:
Sounds good! I'm working on adding in a bunch of tests with all of the new updates now and fixing the pipelines as well.